Gemfile

```diff
@@ -23,6 +21,7 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'awesome_print', '~> 1.6.1'
```

Comment:
You may want to put awesome_print in your development gems if it wouldn't be needed for your production (aka Heroku) deployment